research/00-introduction.tex

```diff
@@ -11,8 +11,8 @@ \section*{Introduction}
 When developing new domain-specific languages using Spoofax, the
 ability for rapid prototyping using a read-eval-print loop (REPL)
 would be very convenient. This report describes the initial research
-phase for this project: extending Spoofax with generating REPLs for the
-languages defined with it.
+phase for this project: extending Spoofax with a REPL that works with
+all Spoofax defined languages.
```

Comment:
No one that notices that there would be two virtually identical sentences in the introduction then?